apps/gi-frontend/tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "types": ["vite/client", "vitest"],
     "module": "ESNext",
     "resolveJsonModule": true,
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "exactOptionalPropertyTypes": false
```

Comment:
Why?

Reply:
with `exactOptionalPropertyTypes` on true, a lot of properties needs to be super defined:
Like you cant do 
```
type Prop={
  something?:string
}
function Com({something}:Prop){...}

// will typeerror out saying string|undefined can't be used
<Com something={cond?"string":undefined} />
```

you will have to explicitly type it to be 
```
type Prop ={
  something?:string|undefined
}
```

Reply:
If you think that level of safety isn't needed then I guess it's fine to remove

Reply:
I'll see if i can refactor this option away. I do agree that this is technically more safety, but I remember i struggled a lot with this when i move the components over. (A lot of libs don't have this option enabled when they wrote their typings, which makes "legal" code have red squiggles)

Reply:
yeah, looks like mui does not support this option. I think any lib with mui components will need this. option